### ISSUE TITLE
Loading External Models - Rendering Strange

### ISSUE THREAD

**Issue by JaimeStill** [looeee/discoverthree.com-examples#3]
I've been following along in StackBlitz building out the examples in Angular. Everything has worked extremely well, up until I got to the *Loading External Models* section. It loads and animates just fine, however the position of the models, and the camera behavior is really strange: [StackBlitz - App](https://three-ng-external-models.stackblitz.io/).

Here's the source: [StackBlitz - SceneService](https://stackblitz.com/edit/three-ng-external-models?file=src%2Fapp%2Fservices%2Fscene.service.ts).

Really looking forward to the completed book!

`three.js version: 0.104.0`.

**Comment by looeee** [comment]
You're loading the models from the three.js repo, and they are not the same as the ones I'm using since I fixed a couple of things.

In particular, I scaled them down to be roughly the size of actual birds (important when using physically correct lighting). The ones on the three.js repo (which you are loading) are about 300 meters wide.

Just scale the models down and they will look the same. I don't remember the exact scale factor I used, but try:

`model.scale.set( 0.05, 0.05, 0.05 )`

I'll add a note to the book mentioning this, thanks for the feedback.
